@types/phaser3-rex-plugins/templates/ui/ui-plugin.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'phaser3-rex-plugins/templates/ui/ui-plugin' {
+  import * as Phaser from 'phaser';
+  import TextEdit from 'phaser3-rex-plugins/plugins/behaviors/textedit/TextEdit';
+  import ObjectFactory from 'phaser3-rex-plugins/templates/ui/ObjectFactory';
+
+  type preTest = (gameObject: Phaser.GameObjects.GameObject, x: number, y: number) => void
+  type postTest = (gameObject: Phaser.GameObjects.GameObject, x: number, y: number) => void
+
+  export default class UIPlugin extends Phaser.Plugins.ScenePlugin {
+    constructor(scene: Phaser.Scene, pluginManager: Phaser.Plugins.PluginManager);
+    add: ObjectFactory
+    getParentSizer(gameObject: Phaser.GameObjects.GameObject): Phaser.GameObjects.GameObject
```

Comment:
This types are reused in many places, could you create a file `plugins/gameobjects/containerlite/GetParent.d.ts` and put there
```
type GetParentFunc = (gameObject: Phaser.GameObjects.GameObject): Phaser.GameObjects.GameObject
```
Similar suggestion for other types here.